src/rail/estimation/algos/deepdisc.py

```diff
@@ -224,7 +221,13 @@ def run(self):
             ),
         )
 
-        self.model = dict(nnmodel=self.model)
+        cfg = LazyConfig.load(self.config.cfgfile)
+        model = instantiate(cfg.model)
+        weights_file_path = os.path.join(self.config.output_dir, self.config.run_name + ".pth")
+        fv_cp = Checkpointer(model, self.config.output_dir)
+        weights = fv_cp._load_file(weights_file_path)
+
+        self.model = dict(nnmodel=weights)
```

Comment:
Here in the `Informer` we load the config file, instantiate the model and create a `Checkpointer` instance to load the model weights. It's likely that this is overkill because `_load_file` really just calls `torch.load(...)` but I did notice that in various versions of fvcore, the parameters passed to `torch.load()` were different. So rather than hardcoding the call to torch, I'll let fvcore maintain that and use the underscore method, `_load_file`.